### PR TITLE
v1.1.0 bump + release notes + docs

### PR DESCRIPTION
**Pull request by fushar**

### CODE REVIEW

File: docs/api-ref/api-ref.rst [hunk 1]
```diff
@@ -118,9 +167,10 @@ The following macros are exposed to define input/output formats:
         void InputFormat() {
             LINES(V) % SIZE(2);
             LINES(X, Y) % SIZE(N);
+            LINES(Z);
         }
 
-    With **V** = {1, 2}, **X** = {100, 110, 120}, **Y** = {200, 210, 220}, **N** = 3, the above format will produce:
+    With **V** = {1, 2}, **X** = {100, 110, 120}, **Y** = {200, 210, 220} **N** = 3, Z = {1, 2, 3, 4} the above format will produce:
```
Comment (jonathanirvings) [inline docs/api-ref/api-ref.rst]: Change to ``**Z**`` for consistency?

File: docs/topic-guides/io-formats.rst [hunk 2]
```diff
@@ -43,9 +43,37 @@ The list of supported segments is given below. In all segments, it must be noted
...
+Tokenized segments
+******************
+
+These segments do some formatting to the value of the variables. For example, the elements of a vector variable in a line will be printed space-separated.
+
+.. note::
+
+    In tokenized segments, string variables cannot have whitespaces. For example, ``hello, world!`` is considered to have two string variables: ``hello,`` and ``world!``. Use raw segments if you want to work with strings containing whitespaces.
+
 Singe line
```
Comment (jonathanirvings) [inline docs/topic-guides/io-formats.rst]: Single?

File: docs/api-ref/api-ref.rst (hunk 1 above)
Comment (fushar) [inline docs/api-ref/api-ref.rst]: Done.

**Comment by jonathanirvings** [comment]
LGTM

**Pull request merged**